tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/commons/css/styles.css

```diff
@@ -1674,3 +1674,134 @@ div.dataTables_wrapper div.dataTables_info {
     padding: 7px;
     border-left: 1px solid #494949;
 }
+
+/*model extra classes*/
+
+.extension > h4 {
+    font-weight: bold;
+    color: #b7b7b7;
+}
+
+div.extension-description > p > code,
+div.extension-description > p {
+    font-size: 17px;
```

Comment:
Standardise font size, seems it common to all extension div

Reply:
848bbc07f1a4d47336d5ee141b19c5ee5207128a this commit fix above need.